.github/workflows/main.yml

```
@@ -173,6 +174,32 @@ jobs:
             OS-${{ runner.os }},
             VM-${{ matrix.platform }},
             Py-${{ steps.python-install.outputs.python-version }}
+            
+  check_changed_folders:
```

Can we use kebab-case for the job id? I've found that it helps distinguish between scripting lang identifiers vs. YAML definition.
```suggestion
  check-changed-folders:
```